### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by brunny-eth**
Create a Contributions guideline file; still needs work but opening it up for review/feedback

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+## Contributing to Pluto
+
+Thank you for your interest in improving Pluto.
+
+There are multiple opportunities to contribute to Pluto, at any level. It doesn't matter if you are just getting started with applied cryptography or if you are an expert; we can use your help. No contribution is too small and we value all contributions.
+
+This document will help you get started. It should be considered as a guide to help you navigate the contribution process. The [Pluto telegram](https://t.me/pluto_xyz/1) is available to help you with any questions concerns that you may have that are not covered in this guide.
+
+If you contribute to this project, your contributions will be made to the project under the Apache 2.0 license.
```
Comment (0xJepsen) [inline CONTRIBUTING.md:27]: link the apache 2.0 license

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+When filing a new bug report in the issue tracker, you will be presented with a basic form to fill out.
+
+If you believe that you have uncovered a bug, please fill out the form to the best of your ability. Do not worry if you
+cannot answer every detail, just fill in what you can. Contributors will ask follow-up questions if something is
+unclear.
+
+The most important pieces of information we need in a bug report are:
+
+- The version that you are on (and that it is up to date)
```
Comment (0xJepsen) [inline CONTRIBUTING.md]:
```suggestion
- The version of the software that you are on (and that it is up to date)
```

**Review by 0xJepsen — APPROVED** [review]
@bruno i really like this. No main criticism here. I think it could be valuable for us to embrace [tiger style](https://github.com/tigerbeetle/tigerbeetle/blob/main/docs/TIGER_STYLE.md)
as it is something a few of us have been really inspired by. I recommend [this video](https://www.youtube.com/watch?v=w3WYdYyjek4) if you have time and are curious about this style of writing software. I would love to hear your thoughts.

File: CONTRIBUTING.md (hunk 1 above)
Comment (brunny-eth) [reply CONTRIBUTING.md:27]: Added the license as a file to this repo and to all other public repos

**Pull request merged**